src/plots/template_attributes.js

```diff
@@ -33,9 +50,9 @@ function describeVariables(extra) {
         }
         descPart = descPart + 'Finally, the template string has access to ';
         if(keys.length === 1) {
-            descPart = 'variable ' + quotedKeys[0];
+            descPart = descPart + 'variable ' + quotedKeys[0];
```

Comment:
Note: I fixed this bug in the `describeVariables()` function, which is the reason for the large number of changes in `plot-schema.json`, since a number of `hovertemplate` and `texttemplate` docstrings were modified. In all cases I think the new string is an improvement.

Reply:
Good catch. Thank you for the fix!